benchmarks/benchmarks/optimize_linprog.py

```diff
@@ -0,0 +1,90 @@
+"""
+Benchmarks for Linear Programming
+"""
+from __future__ import division, print_function, absolute_import
+
+# Import testing parameters
+try:
+    from scipy.optimize import linprog
+    from scipy.linalg import toeplitz
+    from scipy.optimize.tests.test_linprog import lpgen_2d
+    from scipy._lib._numpy_compat import suppress_warnings
+    import numpy as np
+    import os 
+except ImportError:
+    pass
+
+from .common import Benchmark
+
+methods = ["simplex","interior-point"]
```

Comment:
Moved the file to my desktop, `autopep8`, then back again. Fixed this and the other main files.